src/wayscript/secret_manager.py

```diff
@@ -0,0 +1,20 @@
+from urllib.error import HTTPError
+
+from . import utils, context, errors
+
+def set_secret(secret_key: str, secret_val: str) -> bool:
```

Comment:
Are you sure you want this to return a bool since it is always true?

Reply:
haha, used to a language where exceptions Didn't Exist so returning True was for successes and False for failure, hence typing this sort of operation as boolean...

Changed to None